source/require-playgrounds.txt

```diff
@@ -0,0 +1,62 @@
+.. _vsce-require-landing:
+
+============================
+Use require() in Playgrounds
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use the `require()
+<https://nodejs.org/api/modules.html#modules_require_id>`__ function in
+your playgrounds to reuse code in your playgrounds. The code you
+include with ``require()`` can be either:
+
+- A native Node module (such as `fs
+  <https://nodejs.org/api/fs.html#fs_file_system>`__)
+
+- An external Node module downloaded from `npm
+  <https://www.npmjs.com/>`__
+
+- A local JavaScript file
+
+Use Cases
+---------
+
+Using ``require()`` to import code helps keep your playgrounds
+organized. If you are often performing the same tasks in your
+playgrounds, you can use ``require()`` to reuse code and avoid writing
+full functions in each of your playgrounds.
+
+For example, consider the following scenarios where reusing code can
+simplify logic and improve productivity:
+
+- You need to ensure that documents are inserted with a consistent
+  structure. You can import a JavaScript file to validate the data
+  format before the playground script performs the insert.
+
+- You store monetary data and want to convert values between different
+  currencies. You can import a JavaScript function to perform conversion
+  calculations.
+
+- You need to import data from files on your local computer. You can use
+  the native Node module ``fs`` to read data from local files.
```

Comment:
Just dropping a comment to say that I've seen the Vale suggestions but prefer the current wording.

Reply:
Agreed